ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java

```diff
@@ -241,15 +240,7 @@ public void requestBannerAd(Context context,
             return;
         }
 
-        //workaround for missing onPause/onResume/onDestroy

```

Comment:
@HuFangshuai why did you remove this? Did callbacks start to work on all android APIs?

Reply:
@alexander-gubov 

> @HuFangshuai why did you remove this? Did callbacks start to work on all android APIs?
Thanks your comments. I tested on API 28 & 29  and callbacks works well. I am working on testing on more Android APIs.